### DISCUSSION TITLE
Adapting to use with an FTDX-3000 / LP-PAN 2 / Win4YasueSuite

### DISCUSSION THREAD

**Discussion by msteele999** [AlexandreRouma/SDRPlusPlus#215] [General]
I'm currently using an LP-PAN 2 panadapter connected to my Yaesu FT-DX 3000 via Win4YaesuSuite - do you know of any way of interfacing your SDR++ with this setup?

**Comment by AlexandreRouma** [comment]
Hello,

You should be able to use the SoapySDR source (if soapysdr is installed) to use your soundcard as input.

Regards,
Alex

**Comment by msteele999** [comment]
Thank you for the response.

Unfortunately soapysdr doesn't run on windows yet according to their
website.

**Reply by AlexandreRouma** [reply]
It's false, it runs just fine.

Just install it with [PothosSDR](https://downloads.myriadrf.org/builds/PothosSDR/) (make sure to check the "add to path option")
